db/migrate/20240430034154_add_not_null_constraints_and_remove_defaul_values_from_users.rb

```diff
@@ -0,0 +1,15 @@
+class AddNotNullConstraintsAndRemoveDefaulValuesFromUsers < ActiveRecord::Migration[7.1]
+  def change
+    change_column_null :users, :first_name, false
+    change_column_null :users, :last_name, false
+    change_column_null :users, :gender, false
+    change_column_null :users, :date_of_birth, false
+    change_column_null :users, :role, false
+    change_column_null :users, :email, false
+    change_column_null :users, :phone_number, false
+    change_column_null :users, :password_digest, false
+
+    change_column_default :users, :role, from: 2, to: nil
+    change_column_default :users, :gender, from: 0, to: nil
```

Comment:
ditto

Reply:
Why do you write another migration instead of changing previous migrations?

Reply:
@sku-wtag Bhai, I already have some new migrations applied after that specific migration in some other branches but in the same database, so if I change the existing migration file,
- The changes might not be applied because latest applied migration version is higher than the changed one,
- Might face trouble during rollback.

Also, I found on several articles and tutorials that writing new migration files instead of modifying existing ones is better practice. Let me know your thoughts on this.